### PR TITLE
BVAL-512 Provide a way to include sources from the API in the spec

### PR DESCRIPTION
**Pull request by gsmet**
* https://hibernate.atlassian.net/projects/BVAL/issues/BVAL-512

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -42,12 +44,35 @@
...
 		<mkdir dir="target/asciidocextensions" />
 	</target>
+	<target name="validation-api-dir-exists">
+		<delete dir="${validation-api-source-dir}/javax/" />
+		<mkdir dir="${validation-api-source-dir}" />
+	</target>
 
+	<!-- we update the dependency every time we build as it might have changed on disk -->
+	<target name="detect-validation-api-in-m2">
+		<available file="${user.home}/.m2/repository/javax/validation/validation-api/${bv.version}/validation-api-${bv.version}-sources.jar" property="validation-api-in-m2"></available>
```
Comment (gunnarmorling) [inline build.xml]: I'd extract a var at the top of the file for the repo location so one can easily adapt it if needed.

**Comment by gunnarmorling** [comment]
Very nice, just added one comment. Feel free to merge yourself then. Thx!

**Comment by gsmet** [comment]
Pushed a new commit addressing your comment.

**Comment by gunnarmorling** [comment]
Rebased and applied. Thx, @gsmet!

**Pull request closed without merging**